docs/source/integrations/cvat.rst

```diff
@@ -0,0 +1,986 @@
+
+    import fiftyone as fo
+    import fiftyone.zoo as foz
+
+    # Step 1: Load your data into FiftyOne
+    dataset = foz.load_zoo_dataset("quickstart")
+
+    # Step 2: Find a subset of data requiring annotation
+    from fiftyone import ViewField as F
+
+    results = dataset.evaluate_detections(pred_field="predictions", eval_key="eval")
+    high_conf_view = dataset.filter_labels(
+        "predictions", 
+        (F("confidence") > 0.8) & (F("eval") == "fp"),
+    )
+
+    # Step 3: Create a view of samples to annotate
+    # Select a sample for annotation for this example
+    annot_view = high_conf_view.limit(1)
+
+    # Step 4: Send samples to CVAT
+    info = annot_view.annotate(label_field="ground_truth", launch_editor=True)
+
+    # Step 5: In CVAT, annotate samples and save
+
+    # Step 6: Load updated annotations back into FiftyOne
+    annot_view.load_annotations(info, delete_tasks=True)
+
+
+
+CVAT Overview 
+-------------
+
+`CVAT <https://github.com/openvinotoolkit/cvat>`_ is an open-source annotation software for images and videos. 
+It can either be used through the hosted server at `cvat.org
+<https://cvat.org>`_ or through a 
+`custom installation and self-hosted server. <https://openvinotoolkit.github.io/cvat/docs/administration/basics/installation/>`_
+No matter the server being used, an account will be required and the username
+and password must be provided to FiftyOne.
+
+CVAT uses three levels of abstraction for annotation workflows: projects,
+tasks, and jobs. A job contains one or more images and can be assigned to a
+specfic annotator or reviewer. A task defined the label schema to use for
+annotation and contains multiple jobs. A project also allows for a label schema
+and contains multiple tasks.
+
+This integration provides and API to create tasks and jobs, upload data,
+defined label schemas, and download annotations all through Python and
+using FiftyOne. 
+
+When uploading existing labels to CVAT, the unique label ids are uploaded as
+attributes in order to keep track of which labels have been modified, added, or
+deleted. Changing these label ids will result in labels being overwritten when
+loaded into FiftyOne rather than being merged.
+
+
+Setup
+_____
+
+
+Server URL
+----------
+
+Both `cvat.org <https://cvat.org>`_ and custom CVAT servers are supported. The
+following attributes allow specification of the server URL that you have an
+account on and to which you want to upload data:
+
+* `url`: base url of the CVAT server (e.g. `cvat.org` or `localhost`)
+* `port`: four digit port of the custom CVAT server if applicable
+* `https`: boolean indicating whether the URL is `https` (`True`) or `http`
+  (`False`)
+
+
+The environment variables `FIFTYONE_CVAT_URL`, `FIFTYONE_CVAT_PORT`, and
+`FIFTYONE_CVAT_HTTPS` can be set to avoid providing them in every 
+:meth:`annotate() <fiftyone.core.collections.SampleCollection.annotate>`
+call or they can be set in the `~/.fiftyone/annotation_config.json`.
```

Comment:
Link to how to construct an `annotation_config.json`?

Reply:
yeah, documentation should be added to the FiftyOne config user guide page on this, too